docs/src/man/getting_started.md

```diff
@@ -23,6 +23,21 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function can be used to extract the value from the `Nullable` wrapper.
+
+example :
+```julia
+julia> a = Nullable{String}("14:00:00")
+Nullable{String}("14:00:00")
+
+julia> c = get(a)
```

Comment:
I meant adding a sentence pointing to the Julia manual for more information (the URL shouldn't appear, it should just be a link).